apps/ejabberd/src/ejabberd_ctl.erl

```diff
@@ -62,6 +62,8 @@
 -type format_type() :: binary() | string() | char().
 -type cmd() :: {CallString :: string(), Args :: [string()], Desc :: string()}.
 
+-define(ASCII_SPACE_CHARACTER, $\s).
```

Comment:
It's worth using here too: https://github.com/esl/MongooseIM/pull/500/files#diff-5378040ac449fa544bb91213c8dd41e5R394